README.md

```diff
@@ -54,6 +54,16 @@ The key features of Lance include:
 pip install pylance
 ```
 
+To install a preview release:
+
+```shell
+pip install --pre --extra-index-url https://pypi.fury.io/lancedb/ pylance
+```
+
+> [!TIP]
+> Preview releases are released more often than full releases and contain the
+> latest features and bug fixes. They receive the same level of testing as full releases.
```

Comment:
Are we sure fury's capacity is unlimited?  Should we mention here that preview releases may be deleted / cleaned up at some point in the future?

Reply:
They have no documented limit, and I'm pretty sure PyArrow doesn't delete any of their nightlies. But to be safe, I've left a comment sayig we don't guarantee their existance past 6 months.